<ui version="4.0" >
 <class>ConditionsWidget</class>
 <widget class="QWidget" name="ConditionsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <item>
    <widget class="QTreeWidget" name="tree" >
     <property name="headerHidden" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QPushButton" name="new_button" >
       <property name="text" >
        <string comment="new condition">New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="edit_button" >
       <property name="text" >
        <string>Edit...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_button" >
       <property name="text" >
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
